tools/topology/topology1/sof-imx8-virtual.m4

```diff
@@ -0,0 +1,72 @@
+
```

Comment:
This topology seems to be quite generic, not specific to 8MP.
@dbaluta maybe you can test this on 8QM or 8QXP so we can be sure it works on other platforms as well.

Reply:
@SurajSonawane2415 since this is virtual it may be possible to create a ztest for this feature (in another PR), this would mean the feature would be tested by CI. i.e. a small pipeline with virtual loopback as a ztest.

Reply:
@lgirdwood something like this ``zephyr/tests/drivers/i2s/i2s_api/src/test_i2s_loopback.c`` but for DAI, right?

Reply:
Fwiw, this would allow a higher level ztest where we could have a small loopback pipeline and test that we can copy A -> B, this would invoke a lot of pipeline logic/infra and could be easily run in CI. Depends if time permits for development though.